source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```diff
@@ -55,103 +97,127 @@ content: |
         :tabid: java-sync
 
         .. code-block:: java
-           :emphasize-lines: 7-9, 11
 
-           BsonString masterKeyRegion = new BsonString("<Master Key AWS Region>"); // e.g. "us-east-2"
-           BsonString awsAccessKeyId = new BsonString("<IAM User Access Key ID>");
-           BsonString awsSecretAccessKey = new BsonString("<IAM User Secret Access Key>");
+           // TODO: check correctness
```

Comment:
note: todo still here

Reply:
Thanks for spotting this. It's intentional as those will be filled out in other PRs.